app/src/debug/google-services.json

```diff
@@ -62,23 +62,23 @@
       "client_info": {
```

Comment:
Hmmm...looks like there is more than one google-services.json file in the project? If so, I'm guessing we should delete the `debug` one, and just put ours in `main`. I'm guessing Shuttle has multiple Firebase accounts set up, one for debuggin and another for release. For our purposes, I think we only need one.

Reply:
Yes, you're right. One of them is for debugging. However, the devDebug build variant won't compile without the debug version of google-services.json

Reply:
If you delete all google-services.json from the project, and then just put one file under the `app` directory, does that work?

See https://github.com/OneBusAway/onebusaway-android/tree/master/onebusaway-android for an example.

All the build variants should then inherit it.

Reply:
I tried doing that but the project failed to build without the google-services.json in the debug module.

Reply:
Strange - that should work. The debug flavor is the one that failed to build? You may need to do "File->Invalidate caches/restart" to get all the build flavors to pick up the new config.

Reply:
I tried invalidating caches too, but it is still asking for the other json file in the debug.

Reply:
Ok - I guess just duplicate the same file there and see if that works...

Reply:
Yes - It works when I duplicate the debug json file and place it in the app folder.

Reply:
Ok - can you update this PR with that config that works?

Reply:
Sure. Will do.